src/pip/_internal/cli/cmdoptions.py

```diff
@@ -776,27 +776,27 @@ def _handle_no_use_pep517(
     parser.values.use_pep517 = False
 
 
-use_pep517 = partial(
+use_pep517: Any = partial(
```

Comment:
I wonder why this (and the one below) is `Any` instead of `Callable[..., Option]`.

Reply:
Well, even if this were "wrong", Mypy won't fail over there, because it is not on "strict" mode, remember #10091 and #10076.

Reply:
Yeah, I'm more like trying to get into the mind of the person who wrote the original annotation (because all others are `Callable[..., Option]`, persumably they should know this is possible).